.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -1,14 +1,23 @@
 ---
 name: Bug Report
-description: Report a bug encountered while operating Etcd
+description: Report a bug encountered while operating etcd
 labels:
   - type/bug
 body:
+  - type: checkboxes
+    id: confirmations
+    attributes:
+      label: Bug report criteria
+      description: Please confirm this bug report meets the following criteria.
+      options:
+        - label: This bug report is not security related, security issues must be disclosed privately via security@etcd.io.
+        - label: This is not a support request, support requests must be raised in the etcd [discussion forums](https://github.com/etcd-io/etcd/discussions).
+        - label: You have read the etcd [bug reporting guidelines](https://github.com/etcd-io/etcd/blob/main/Documentation/contributor-guide/reporting_bugs.md).
+        - label: Existing open issues along with etcd [frequently asked questions](https://etcd.io/docs/latest/faq) have been checked and this is not a duplicate.
```

Comment:
```suggestion
        - label: This bug report is not security related, which should be disclosed privately via security@etcd.io.
        - label: This is not a support request, which should be raised in the etcd [discussion forums](https://github.com/etcd-io/etcd/discussions).
        - label: You have read the etcd [bug reporting guidelines](https://github.com/etcd-io/etcd/blob/main/Documentation/contributor-guide/reporting_bugs.md).
        - label: Existing open issues along with etcd [frequently asked questions](https://etcd.io/docs/latest/faq) have been checked and this is not a duplicate.
```

Reply:
I think duplication here makes sense as it makes the working stronger and explicit.

Reply:
Please feel free to ignore the duplication. 

I feel like `must` is a little harsh :) , leave to your guys to decide.

Reply:
Kept the duplication but switched to `should`.  Middle ground.  We can always tweak it later if we find there are still too many issues being created incorrectly 🙂